core/domain/wipeout_service_test.py

```diff
@@ -5018,6 +5018,16 @@ def test_repeated_deletion_is_successful_when_emails_disabled(self):
             wipeout_service.delete_users_pending_to_be_deleted()
             self.assertEqual(len(self.email_bodies), 0)
 
+    def test_no_email_is_sent_when_there_are_no_users_pending_deletion(self):
```

Comment:
Can you show (e.g. via a screenshot in your PR opening comment) that this fails when expected (i.e. then the fix is not in place)?

Reply:
I've added a screenshot above that shows this test catches a regression in this functionality. PTAL!

Reply:
Thanks!